src/clients/HubPoolClient.ts

```diff
@@ -281,13 +281,12 @@ export class HubPoolClient extends BaseAbstractClient {
    * @param chainId Chain where the `tokenAddress` exists in TOKEN_SYMBOLS_MAP.
    * @returns Token info for the given token address on the L2 chain including symbol and decimal.
    */
-  // TODO: didn't change `tokenAddress` here to Address because of downstream getTokenInfo impl
```

Comment:
@bmzig Do you recall any more about what the issue was here? I didn't see any major issues in changing the interfaces; was there potentially a chicken-and-egg issue with the relayer repo?

Reply:
Resolved in DM - should be no blockers for this.